CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -362,6 +362,9 @@ strCounterHiddenDarkEventDescription="The Resistance heard about a <font color='
 strLabel=View Chain
 
 [UIUtilities_InfiltrationTutorial]
+strWelcomeHeader=Welcome
+strWelcomeBody="<Bullet/> Welcome to Covert Infiltration. As you progress, you will be greeted with tutorial popups which explain the changes that the mod makes. Take your time to read them carefully. They explain all the major aspects you need to know to effectively make decisions. In case you want to review the tutorials later, you can always find them in the XCOM Archives in the Commander�s Quarters. <br/><Bullet/> If you want to, you can disable the tutorials in the MCM. In that case, you can still find them in the Archives, where all of them will be unlocked irrespective of where you are in the campaign. <br/><Bullet/> Note that the tutorials will not adapt if you edit any of the Strategy Overhaul mods' core values in the configuration files. If you have made any major changes or removals to suit your personal taste, the tutorials may be outdated."

```

Comment:
There's a broken apostrophe in "Commander's Quarters", denoted by the diamond with the ? symbol. Copy pasting certain characters into ModBuddy breaks them, delete the broken character and retype it in ModBuddy to fix.